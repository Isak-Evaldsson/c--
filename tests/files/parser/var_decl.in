int func1 () {
    int a;
}

int func2 (int a) {
    int b = 0;
}

int func3 () {
    int c;
    int d = 7;
}