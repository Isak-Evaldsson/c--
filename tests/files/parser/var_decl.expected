Prototype: func1
    VarDecl: a
Prototype: func2
    Param: a
    VarDecl: b
        LiteralExpr: 0
Prototype: func3
    VarDecl: c
    VarDecl: d
        LiteralExpr: 7
